CMakeLists.txt

```diff
@@ -215,6 +215,9 @@ if("${LINKER_VERSION}" MATCHES "GNU gold" OR "${LINKER_VERSION}" MATCHES "GNU ld
 else()
   message(STATUS "Using unknown linker, not setting linker optimizations")
 endif ()
+if (STATIC_LIBSTDCXX)
+  set(LINKER_FLAGS "${LINKER_FLAGS} -static-libstdc++")
+endif()
```

Comment:
CMake picks up your environment's `CXXFLAGS` and `LDFLAGS` --- there should be no need for this switch here, just export the appropriate flags.